v2/stack_analyses.yaml

```diff
@@ -557,6 +564,21 @@ components:
           registered: '#/components/schemas/StackAnalysesResultForRegisteredUser'
           freetier: '#/components/schemas/StackAnalysesResultForFreeTier'
       properties:
+        version:
+          type: string
+          example: v2
+        started_at:
+          type: string
+          example: "2020-04-06 11:56:08.349405"
+        ended_at:
+          type: string
+          example: "2020-04-06 11:58:18.349405"
+        external_request_id:
```

Comment:
Isnt this also part of the request?

Reply:
The place where you POST the request(vs extension) and the place where you GET the response is different. So it is good to have id as part of the response.

Reply:
Removed it